src/utils/generatePrompt.ts

```diff
@@ -91,5 +93,46 @@ export const modifier = (wordLimit: number) => {
   if (wordLimit < 50) return "an extremely short";
   if (wordLimit < 100) return "a short";
   if (wordLimit < 200) return "a very detailed";
-  return "multiple paragraphs of an incredibly long, detailed, and verbose and longwinded text in the style of a new york times article";
+  return "multiple paragraphs of an incredibly long, detailed, and verbose and long-winded text in the style of a new york times article";
 };
+
+/* ------ Chat GPT Prompts below ------ */
+
+// As of March 8, 2023, chatGPT api docs recommends setting the system role to user to set more explicit instructions as sometimes the system prompt is ignored by some models
+
+// https://platform.openai.com/docs/guides/chat/instructing-chat-models
+
+export function generateCondensedSummaryPromptObjectArray(text: string, wordLimit: number): ChatGPTPromptPropsItem[] {
```

Comment:
No we don't, I added the user role instead of system to be more explicit (see comment above). As I was refining the prompt I noticed that sometimes the user role would be referenced in the result for some prompts so I changed it to system. For consistency's sake I'll just make it a system role here and leave the comment.